src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -223,30 +205,50 @@ else if (!StringUtils.isNullOrEmpty(setup))
                 dtlsStack.actAsServer();
             }
         }
+    }
 
-        super.startConnectivityEstablishment(transportPacketExtension);
+    public void startDtlsHandshake()
+    {
+        logger.info("Starting DTLS.");
+        try
+        {
+            if (dtlsStack.getRole() == null)
+            {
+                logger.warn("Starting the DTLS stack before it knows its role");
+            }
+            dtlsStack.start();
+        }
+        catch (Throwable e)
+        {
+            logger.error("Error during DTLS negotiation: " + e.toString() +
+                    ", closing this transport manager");
+        }
     }
 
     /**
      * Returns {@code true} if this {@link DtlsTransport} is connected. It is
      * considered connected if the underlying ICE connection has been
```

Comment:
Is the comment about the "underlying ICE connection" still correct/relevant here?

Reply:
Nope, fixed